src/test/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverterTest.java

```diff
@@ -161,15 +163,23 @@ public void convertContentPackage() throws Exception {
                     .setEmitter(DefaultPackagesEventsEmitter.open(outputDirectory))
                     .convert(packageFile);
 
+            List<ContentPackageExtensionVerifier> verifiers = new ArrayList<>();
+            verifiers.add(new ContentPackageExtensionVerifier("asd.sample:asd.retail.apps:zip:cp2fm-converted:0.0.1"));
+            verifiers.add(new ContentPackageExtensionVerifier("asd.sample:Asd.Retail.ui.content:zip:cp2fm-converted:0.0.1"));
+
+            ContentPackageExtensionVerifier configPackageVerifier = new ContentPackageExtensionVerifier("asd:Asd.Retail.config:zip:cp2fm-converted:0.0.1");
+
+            configPackageVerifier.setExpectedFilterXml(expectedConfigPackageFilter);
+            verifiers.add(configPackageVerifier);
+            verifiers.add(new ContentPackageExtensionVerifier("asd.sample:asd.retail.all:zip:cp2fm-converted:0.0.1"));
+
             verifyFeatureFile(outputDirectory,
                             "asd.retail.all.json",
                             "asd.sample:asd.retail.all:slingosgifeature:0.0.1",
-                            Collections.singletonList("org.apache.felix:org.apache.felix.framework:6.0.1"),
-                            Collections.singletonList("org.apache.sling.commons.log.LogManager.factory.config~asd-retail"),
-                            Arrays.asList("asd.sample:asd.retail.apps:zip:cp2fm-converted:0.0.1",
-                                            "asd.sample:Asd.Retail.ui.content:zip:cp2fm-converted:0.0.1",
-                                            "asd:Asd.Retail.config:zip:cp2fm-converted:0.0.1",
-                                            "asd.sample:asd.retail.all:zip:cp2fm-converted:0.0.1"));
+                            Arrays.asList("org.apache.felix:org.apache.felix.framework:6.0.1"),
+                            Arrays.asList("org.apache.sling.commons.log.LogManager.factory.config~asd-retail"),
```

Comment:
changing from Collections.singletonList to Arrays.asList seems to be an unrelated change. please revert if it is not required for the test to pass